### ISSUE TITLE
Issue:  Plain "www." prefixed URLs are not detected or clickable.

### ISSUE THREAD

**Issue by jtbrown3** [jfcherng/Sublime-OpenUri#12]
Hello,  Thx for publishing this excellent plugin/package for Sublime Text!
Unfortunately, the plugin/package is currently not detecting URLs prefixed with only "www." (whereas right-clicking in Sublime Text 3/4 DOES detect plain "www" links)

For example, OpenUri will not detect simple links like:

- www.microsoft.com
- www.cnn.com
- www.coalescence.global (my company site)

The above links are all detected properly by default when right-clicked within Sublime Text.

Is there any way you could add the capability to detect www links?  (this would make provide consistency with the existing functionality within ST 3/4)

Thanks for considering! -JT

**Comment by jfcherng** [comment]
```py
rex = re.compile(
    r'''(?x)
    \b(?:
        https?://(?:(?:[\w\d\-]+(?:\.[\w\d\-.]+)+)|localhost)|  # http://
        www\.[\w\d\-]+(?:\.[\w\d\-.]+)+                         # www.
    )
    /?[\w\d\-.?,!'(){}\[\]/+&@%$#=:"|~;]*                       # url path and query string
    [\w\d\-~:/#@$*+=]                                           # allowed end chars
    ''')
```

Yes, so ST treats "www" without a scheme as a special case.

---

Added in b1cfb1e2fb7a36838ac06bb1f3c45f216b74c170 and released as v7.2.0.

**Comment by jtbrown3** [comment]
Wow!
Thank you Jack @jfcherng for the amazingly fast enhancement/turnaround on this!
I've already tested and it works great!
Blessings!  -JT 🙏
